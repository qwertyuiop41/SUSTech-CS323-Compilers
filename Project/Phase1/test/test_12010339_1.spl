int main() {
    int num1 = 10;
    it num2 = 20;
    int sum = num1 ? num2;
}