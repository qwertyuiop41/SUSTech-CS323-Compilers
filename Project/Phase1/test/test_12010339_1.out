Error type B at Line 3: Missing specifier
Error type A at Line 4: unknown lexeme ?
